### ISSUE TITLE
News OR Events Container Template to be able to set "hero" tiles

### ISSUE THREAD

**Issue by rhyslawry** [unimelb/matrix-controlled-environment#227]
Would be great to have the ability to set specific blocks to "hero" within the News OR Events container listing. Has been requested a few times, and would be a big feature to convince people to use the code-free templates rather than hard-code news listings (as many currently are).
Hero tiles are available for the combined listing, but for smaller sites the combined listing is unnecessary (overkill).

Could be done two ways:
1. Just like the combined listing, with asset IDs in a text box separated by commas i.e.
hero: <Asset ID>, <Asset ID> etc.

2. Alternatively, have it form part of the metadata schema for local news items, just like the Election Watch site: http://electionwatch.unimelb.edu.au/
All news items had a metadata field:
Homepage - Hero tile: Yes/No (no by default)
![image](https://cloud.githubusercontent.com/assets/15081925/22094908/100e86ac-de65-11e6-8e2d-4a357fda979e.png)

(My personal favourite is no. 2)

**Comment by dkevey** [comment]
Would definitely like to sort out the hero item. Not sure about the second option with additional metadata schema due to the high overhead involved. I actually think the whole news and events listing style needs to be redesigned to differentiate the 2 types of assets, as it can all get a bit muddled at times.

**Comment by rhyslawry** [comment]
I agree, the whole "paginate" number with the combined listing needs to also take into consideration if any blocks were hero, otherwise `paginate: 8` will spill over to a third line if one of them was a hero.
This is obviously easier said than done. But is there some way for hero items to count as "2" when counting out 8 blocks each time the "See more" button is pressed?

Should I submit this as a separate issue seeing as it has to do with the Combined Listing not the News OR Events?

**Comment by dkevey** [comment]
I think that now we have conditional metadata this might be possible to proceed with.
